server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessageTest.java

```diff
@@ -209,6 +210,24 @@ public void generateMessageShouldAddSizeWhenMimeMessageAsSome() throws Exception
                 "\n" +
                 "Message details:\n" +
                 "  MAIL FROM: null\n" +
-                "  Size (in bytes): 6\n");
+                "  Size: 6 B\n");
+    }
+
+    @Test
+    public void generateMessageShouldSpecifySizeInAReadableWay() throws Exception {
+        String content = "MIME-Version: 1.0\r\n" +
+            "Content-Type: text/plain; charset=utf-8\r\n" +
+            "\r\n" +
+            String.join("", Collections.nCopies(1000, "test\r\n"));
+        MimeMessage message = new MimeMessage(Session.getDefaultInstance(new Properties()), new ByteArrayInputStream(content.getBytes()));
+        FakeMail mail = FakeMail.from(message);
+
+        String generateMessage = new NotifyMailetsMessage().generateMessage("my message", mail);
+
+        assertThat(generateMessage).isEqualTo("my message\n" +
```

Comment:
equality is too much, you should just grep for Size:

Reply:
done

Reply:
done